test/ao/sparsity/test_parametrization.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+from torch import nn
+from torch.ao.sparsity import utils
+from torch.nn.utils import parametrize
+
+import torch
+from torch.testing._internal.common_utils import TestCase
+
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', level=logging.INFO)
+
+class ModelUnderTest(nn.Module):
+    def __init__(self, bias=True):
+        super().__init__()
+        self.linear = nn.Linear(16, 16, bias=bias)
+        self.seq = nn.Sequential(
+            nn.Linear(16, 16, bias=bias),
+            nn.Linear(16, 16, bias=bias)
+        )
+
+        # Make sure the weights are not random
+        self.linear.weight = nn.Parameter(torch.zeros_like(self.linear.weight) + 1.0)
+        self.seq[0].weight = nn.Parameter(torch.zeros_like(self.seq[0].weight) + 2.0)
+        self.seq[1].weight = nn.Parameter(torch.zeros_like(self.seq[1].weight) + 3.0)
+        if bias:
+            self.linear = nn.Parameter(torch.zeros_like(self.linear.bias) + 10.0)
+            self.seq[0] = nn.Parameter(torch.zeros_like(self.seq[0].bias) + 20.0)
+            self.seq[0] = nn.Parameter(torch.zeros_like(self.seq[0].bias) + 30.0)
+
+        # Make masks stored in the modules for convenience
+        self.linear.register_buffer('mask', torch.eye(16))
```

Comment:
I dont think you need this in the rest of the code, also since masks are part of the parameterized module, this is confusing. Can you remove this?

Reply:
This is for convenience of the unittests only. The overall flow does not involve this